homeassistant/components/backup/websocket.py

```diff
@@ -0,0 +1,74 @@
+"""Websocket commands for the Backup integration."""
+import voluptuous as vol
+
+from homeassistant.components import websocket_api
+from homeassistant.core import HomeAssistant, callback
+
+from .const import DOMAIN
+from .manager import BackupManager
+
+
+@callback
+def async_register_websocket_handlers(hass: HomeAssistant) -> None:
+    """Register websocket commands."""
+    websocket_api.async_register_command(hass, handle_info)
+    websocket_api.async_register_command(hass, handle_create)
+    websocket_api.async_register_command(hass, handle_remove)
+
+
+@websocket_api.require_admin
+@websocket_api.websocket_command({vol.Required("type"): "backup/info"})
+@websocket_api.async_response
+async def handle_info(
+    hass: HomeAssistant,
+    connection: websocket_api.ActiveConnection,
+    msg: dict,
+):
+    """List all stored backups."""
+    manager: BackupManager = hass.data[DOMAIN]
+
+    if not manager.backups:
```

Comment:
It initializes backups with an empty dict, which is also falsey.

Instead, initialize it as `None` in the constructor, so you know it hasn't been loaded. 

When you create a backup, you don't need to add it either unless `backups is not None` (because it will be loaded when listed for first time)